### DISCUSSION TITLE
ReDoS?

### DISCUSSION THREAD

**Discussion by tamarGaler** [zricethezav/gitleaks#1476] [General]
Hey! 
what is the process that ensure that no rules that can cause ReDoS are introduced into the code base?

**Comment by rgmz** [comment]
[Go's regular expression engine is less susceptible to ReDOS issues](https://michaelhidalgocr.blogspot.com/2019/05/is-it-gos-regexp-package-vulnerable-to.html).

Though I would not consider ReDoS impactful for GitLeak's usecase.
